### DISCUSSION TITLE
Arbitrary Boundary?

### DISCUSSION THREAD

**Discussion by fzmwml** [zwicker-group/py-pde#354] [Q&A]
Hi:

Can one define one's own Boundary Curve, say a triangle or any plygon?

If so, could you please direct me to an example of how to do so?

**Comment by david-zwicker** [comment]
`py-pde` focuses on very simple geometries and thus does *not* allow defining arbitrary boundary curves. You might want to look into finite element or finite volume simulation methods instead.
